app/views/detainees/_form.html.slim

```diff
@@ -33,7 +33,7 @@
           legend_options: { class: 'govuk-label' }
 
       .govuk-form-group
-        label.govuk-label for='auto_ethnicity' Ethnicity
+        label.govuk-label for='auto_ethnicity' Ethnicity #{' (Optional)' if escort.from_police?}
```

Comment:
Probably a better I18n way to do this; `location_text` and `localized_label` come to mind - can't quite remember how they work but I think they address this sort of scenario.